semgrep_metrics.atd

```diff
@@ -60,6 +60,10 @@ type environment = {
     ?rulesHash <ocaml mutable>: sha256 option;
     (* TODO: should be just boolean *)
     ci <ocaml mutable>: string nullable;
+    (* SHA-256 hash of the commit ID specified via `--baseline_commit` option.
+       If `baseline_commit` option is set, it implies that the scan is operating
+       in differential scan mode. Since semgrep 1.46 *)
```

Comment:
But do we actually care about the sha256? What we gonna do with this metrics?
Why not just a boolean like is_diff_scan: bool;?

Reply:
changed to a boolean type. I added it as a hash just in case we ever wanted to distinguish between runs with different `baseline-commit` values specified, but it's probably not worth it.